mlir/lib/Dialect/Tosa/Utils/ConversionUtils.cpp

```diff
@@ -213,3 +213,29 @@ mlir::tosa::convertFromIntAttr(const DenseElementsAttr &attr, const int rank) {
   }
   return {};
 }
+
+Attribute mlir::tosa::getConstantAttribute(Operation *op) {
+  if (op == nullptr) {
+    return Attribute();
+  }
+
+  if (isa<ConstOp>(op)) {
+    return cast<ConstOp>(op).getValues();
+  }
```

Comment:
Nit: this would be more c o n v e n t i o n a l
```suggestion
  if (auto constOp = dyn_cast<ConstOp>(op)) {
    return constOp.getValues();
  }
```

Reply:
Fixed